src/components/QuoteBubble/CopyNotification.js

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+import { useTransition, animated } from 'react-spring';
+
+import { zIndexArtworkPart } from '../../styles/designTokens';
+import { CopyIndicator } from '../Icons';
+
+const Wrapper = styled(animated.div)`
+  position: absolute;
+  top: 1.4rem;
+  left: 1.4rem;
+  transform-origin: 50% 100%;
+  z-index: ${zIndexArtworkPart};
+`;
+
+function CopyNotification({ isVisible, onFinished }) {
+  const transitions = useTransition(isVisible, null, {
```

Comment:
I think that you can do that animation way easier with `useSpring`, and similar logic as we did in `Button.js`, or `ButtonBase.js`

Reply:
According to documentation for **mount/unmount single-component reveals** should be handled by **useTransition**.